src/controllers/todo_lists.controller.js

```diff
@@ -11,74 +10,54 @@ export class ToDoListsController extends BaseController {
    * Constructor
    */
   constructor() {
-    let notFoundMsg = `ToDo List not found`
+    const notFoundMsg = 'ToDo List not found';
 
-    super(notFoundMsg)
-    this.ToDoList = new ToDoList()
+    super( notFoundMsg );
+    this.ToDoList = new ToDoList();
   }
 
   /**
    * Retrieve the list of all ToDo lists
    */
-  index(request, reply) {
-    this.ToDoList.findAll()
-      .then((response) => reply(response))
-      .catch((err) => reply(this.Boom.wrap(err)))
+  index( request, reply ) {
+    this.handleRequest( this.ToDoList.findAll(), reply );
```

Comment:
Anything that can be used by the derived classes it is ok to be defined by name, so this method is not private IMO. Anything internal in the base class (that should not be used by the derived classes) is better to be used by Symbol or WeakMap.